demo/GLCanvas.py

```diff
@@ -62,14 +71,15 @@ def OnButton(self, evt):
             dlg = wx.MessageDialog(self,
                                    'The OpenGL package was not found.  You can get it at\n'
                                    'http://PyOpenGL.sourceforge.net/',
+                                   '/nor $ pip install PyOpenGL PyOpenGL_accelerate',
```

Comment:
```
Traceback (most recent call last):
  File "./GLCanvas.py", line 71, in OnButton
    dlg = wx.MessageDialog(self,
TypeError: MessageDialog(): argument 4 has unexpected type 'str'
```

Reply:
Opps. that one wasn't intended. Guess I had a brain fart. Fixed comma typo and formatting.
Also removed other requested lines.